core/src/main/java/com/linecorp/armeria/server/DefaultPathMappingContext.java

```diff
@@ -40,7 +40,20 @@
 /**
  * Holds the parameters which are required to find a service available to handle the request.
  */
-final class DefaultPathMappingContext implements PathMappingContext {
+public final class DefaultPathMappingContext implements PathMappingContext {
+
+    /**
+     * Returns a new {@link PathMappingContext} instance.
+     */
+    public static PathMappingContext of(VirtualHost virtualHost, String hostname,
```

Comment:
This method could be moved to `PathMappingContext` and `DefaultPathMappingContext` could remain package-local?